managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+Selecting this credential type enables synchronization of cloud inventory with Amazon Web Services.
+
+- **Access Key**: User credentials that allow for programmatic calls to Amazon Web Services.
+
+- **Secret Key**: The secret key that corresponds to the user access key.
+
+- **STS Token**: Token generated by Amazon Web Services Security Token Service.
+
+##### Azure
+
+Selecting this credential type enables synchronization of cloud inventory with Microsoft Azure. Microsoft Azure credentials have several attributes to configure:
+
+- **Username**: The username to use to connect to the Microsoft Azure account.
+
+- **Password**: The password to use to connect to the Microsoft Azure account.
+
+- **Subscription ID**: The Subscription UUID for the Microsoft Azure account.
+
+- **Tenant ID**: The Tenant ID for the Microsoft Azure account.
+
+- **Client Secret**: The Client Secret for the Microsoft Azure account.
+
+- **Client ID**: The Client ID for the Microsoft Azure account.
+
+##### OpenStack
+
+Selecting this credential type enables synchronization of cloud inventory with Red Hat OpenStack Platform. OpenStack credentials have several attributes that can be configured:
+
+- **Username**: The username to use to connect to OpenStack.
+
+- **Password (API Key)**: The password or API key to use to connect to OpenStack.
+
+- **Host (Authentication URL)**: The host to be used for authentication.
+
+- **Project (Tenant Name)**: The Tenant name or Tenant ID used for OpenStack. This value is usually the same as the username.
+
+- **Domain name**: The FQDN to be used to connect to OpenStack.
+
+##### Red Hat Virtualization
+
+Selecting this credential type enables synchronization of cloud inventory with Red Hat Virtualization. Red Hat Virtualization credentials have several attributes that can be configured:
+
+- **Username**: The username to use to connect to Red Hat Virtualization.
+
+- **Password**: The password to use to connect to Red Hat Virtualization.
+
+- **Host (Authentication URL)**: The host to be used for authentication.
+
+   **Important**: Enter in **Host** the Red Hat Virtualization provider URL, followed by the path `/ovirt_engine/api`. Example: `https://your.rhv.com/ovirt_engine/api`
+
+##### VMware
+
+Selecting this credential type enables synchronization of inventory with VMware vCenter.
+
+**Important:** If both {{ site.data.product.title_short }} and a VMware provider are located in the same IPv6-only network, use a DNS-resolvable hostname for the VMware provider in the **vCenter Host** field when adding credentials.
+
+VMware credentials have several attributes that can be configured:
+
+- **Username**: The username to use to connect to vCenter.
+
+- **Password**: The password to use to connect to vCenter.
+
+- **vCenter Host**: The vCenter hostname or IP address to connect to.
+
+**Note:** If the VMware guest tools are not running on the instance, VMware inventory sync might not return an IP address for that instance.
```

Comment:
These are all from Ansible and not used by workflows and can be deleted from here

Reply:
I have removed these types as requested.